app/code/core/Mage/Tax/etc/config.xml

```diff
@@ -298,9 +298,6 @@
                 <full_summary>0</full_summary>
                 <zero_tax>0</zero_tax>
             </sales_display>
-            <notification>
-                <url>http://www.magentocommerce.com/knowledge-base/entry/magento-ce-18-ee-113-tax-calc</url>
```

Comment:
This break the source link here:

![image](https://user-images.githubusercontent.com/31816829/167080142-cc6bd51e-1729-4fca-a6f6-8b7a36e2ecd6.png)

I suggest this url: https://web.archive.org/web/20150315064836/http://www.magentocommerce.com/knowledge-base/entry/magento-ce-18-ee-113-tax-calc
Or perhaps we can copy the page to openmage.org.

Reply:
Perhaps the phrase "Please see source for more details" should be removed from the warning message.

Reply:
@luigifab I've modified the PR adding your link, great idea! I looked on google's cache but coulnd't find anything, I didn't think about archive.org!